### PR TITLE
Reset timeout values for test_scale and test_reconfig

### PR DESCRIPTION
**Pull request by anurag6**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@ jobs:
 
   test_reconfig:
     runs-on: ubuntu-latest
-    timeout-minutes: 40
+    timeout-minutes: 50
```
Comment (grafnu) [inline .github/workflows/main.yaml:67]: are you sure this is needed?  I'm seeing a successful run of this take ~20min, why bump to 50?

**Review by grafnu — COMMENTED** [review]
Thanks for looking at this!  I know test_scale needs it, but I didn't think test_reconfig did... are you sure that's necessary?

File: .github/workflows/main.yaml (hunk 1 above)
Comment (pbatta) [reply .github/workflows/main.yaml:67]: test_reconfig also timed out a few times (https://github.com/faucetsdn/forch/actions/runs/305295427, https://github.com/faucetsdn/forch/actions/runs/305122643).

**Comment by grafnu** [comment]
Yes, but the question is if the increased-with-longer-timeout is a valid
success condition -- or is it wedged indefinitely? Or indicative of a
failure condition?  And/or, why is there such high variance in the success
time?

**Comment by anurag6** [comment]
> Thanks for looking at this! I know test_scale needs it, but I didn't think test_reconfig did... are you sure that's necessary?

Oh I jst wanted to do a sample PR to see if @pbatta hypothesis works.

**Comment by anurag6** [comment]
I am running the tests a few times to see if any of them fails. test_reconfig doesn't seem to be wedged indefinitely, though. It was usually at different points of the run where it was canceled. If it were edged indefinitely, the additional 10 min bump shouldn't help it. We could run the tests a few times and see if it fails again.

**Pull request closed without merging**